docs/api/qiskit-ibm-provider/index.md

```diff
@@ -1,66 +1,16 @@
 ---
```

Comment:
This used to be `ibm-provider.md` (with `-`, not `_`). The only change we've made is the front matter / metadata.